bgpd/bgp_attr.c

```diff
@@ -904,16 +904,20 @@ static void attrhash_finish(void)
 static void attr_show_all_iterator(struct hash_bucket *bucket, struct vty *vty)
 {
 	struct attr *attr = bucket->data;
+	struct in6_addr sid = {};
+
+	if (attr->srv6_l3vpn)
+		sid = attr->srv6_l3vpn->sid;
+	else if (attr->srv6_vpn)
+		sid = attr->srv6_vpn->sid;
 
 	vty_out(vty, "attr[%ld] nexthop %pI4\n", attr->refcnt, &attr->nexthop);
 
 	vty_out(vty,
 		"\tflags: %" PRIu64
 		" distance: %u med: %u local_pref: %u origin: %u weight: %u label: %u sid: %pI6\n",
 		attr->flag, attr->distance, attr->med, attr->local_pref,
-		attr->origin, attr->weight, attr->label,
-		attr->srv6_l3vpn ? &attr->srv6_l3vpn->sid
-				 : &attr->srv6_vpn->sid);
+		attr->origin, attr->weight, attr->label, &sid);
```

Comment:
Will a vty_out with a ipv6 address of all 0's being printed going to make sense?
Wouldn't it be better to output NULL? or Unknown?

Reply:
Will change to `Unset`.

Reply:
Fixed.